src/main/java/mekanism/client/gui/GuiTeleporter.java

```diff
@@ -85,6 +92,11 @@ public void func_231160_c_() {
             }
             updateButtons();
         }));
+        func_230480_a_(new GuiSlot(SlotType.NORMAL, this, 131, 120));
```

Comment:
This should have a `.setRenderAboveSlots()` on the end of the slot creation like the logistical sorter does. (If it doesn't in your code base just pull in/rebase onto the latest 1.16 branch to see what I mean)

Reply:
This was just copied from the QIO, but i will look into that.

Reply:
It may not be strictly needed due to there being no "background" for the slots, but probably is worth doing anyways. (May make sense to adjust the QIO ones to be like it as well).

Reply:
done.